[Docs] Spend cap guide (#12878)
supabase/supabase: apps/docs/pages/guides/platform/spend-cap.mdx

import Layout from '~/layouts/DefaultGuideLayout'

export const meta = {
  title: 'Spend cap',
  description: 'Learn about the cost control of Supabase.',
}

> For managing cost control for each project, there is a Spend Cap to manage your usage and costs.

Reviewer: ```suggestion
Supabase offers a "Spend Cap" on each project to manage your usage and costs.
```

> Each project's Spend Cap determines wether your project can exceed the free quota allowance of any paid [line item](#what-can-i-be-charged-for) in a given billing cycle.

Reviewer: ```suggestion
This Spend Cap determines whether your project can exceed the free quota allowance of any [line item](#what-can-i-be-charged-for) in a given billing cycle.
```

If you exceed the free quota allowance of a line item, and the Spend Cap is **"off"**, then you will be billed for any additional usage on that line item.

## What additional usage can I be charged for?

Supabase charges for certain usage line items in your project. Those line items include:

- Database Size
- Storage Size
- Egress
- Asset transformations

> - Realtime connections

Reviewer 2: ```suggestion
```

We only have the peak connections as a line item, not the connections itself

- Realtime concurrent peak connections
- Realtime messages
- Edge function invocations
- Number of Edge functions

You can view the pricing, and free quota allowance of each line item on the [pricing page](/pricing).

You can monitor your usage and costs using the [Usage section of your project](https://app.supabase.com/project/_/settings/billing/usage) in the dashboard.
This will show you how much you have spent so far in the current billing cycle, as well as an estimate of your expected total cost for the cycle based on your current usage.

## How can I turn the Spend Cap on or off?

To set up a Spend Cap in your project, navigate to the [Subscription section](https://app.supabase.com/project/_/settings/billing/subscription) of your project.
From there, you can click on "Change subscription" to select the plan you want to use and, at the same time, turn on or off the Spend Cap.

> ## What happens when the Spend Cap is on

Reviewer: ```suggestion
## What happens when the Spend Cap is on?
```

Once you use all your free quota allowance for a [line item](#what-can-i-be-charged-for), your project will not allow any further usage on that line item until the next billing cycle begins.

This can help you to avoid unexpected costs and manage your expenses more effectively.

## What happens when Spend Cap is off?

> Once you use all the free quota allowance for a [line item](#what-can-i-be-charged-for), your project will continually be able to make further usage.

Reviewer: ```suggestion
Your project will continue to operate after using all the free quota allowance for each [line item](#what-can-i-be-charged-for).
```

This additional usage will be charged based on the line item's cost per unit outlined on the [pricing page](/pricing).

<Admonition type="note">

>   It is advised, when your Spend Cap is **off**, to monitor your usage and costs using the [Usage

Reviewer: ```suggestion
  When your Spend Cap is **off**, we recommend monitoring your consumption and costs using the [Usage
```
